app/webpacker/components/Tickets/TicketLogs.jsx

```diff
@@ -12,14 +12,14 @@ export default function TicketLogs({ ticketId }) {
     queryFn: () => getLogs({ ticketId }),
   });
 
-  function logText(actionType, actionValue) {
+  function logText(actionType) {
```

Comment:
Why are you getting rid of `actionValue`? You're still storing that information...

Reply:
This is because adding all sub-actions can be complicated. I prefer to do it along with other missing informations (in the log - like acting stakeholder, etc) using a good UI library (like [Feed](https://react.semantic-ui.com/views/feed/)) in the follow-up PR.